serviceerror/convert.go

```diff
@@ -75,9 +75,12 @@ func FromStatus(st *status.Status) error {
 		return newUnavailable(st)
 	case codes.Unimplemented:
 		return newUnimplemented(st)
+	case codes.AlreadyExists:
```

Comment:
This breaks later code that uses already exists below for specific namespace, workflow execution, and cancellation errors. If we must have a generic already-exists, add as an else to those later.

Reply:
Good catch, thanks!